airbyte-webapp/src/components/connection/CatalogTree/StreamFieldTable.module.scss

```diff
@@ -1,8 +1,15 @@
 @use "scss/colors";
 @use "scss/variables";
 
+.container {
+  padding-bottom: variables.$spacing-md;
+
+  // HACK: Hot-fixes an issue in StreamHeader SCSS where the .catalogSectionRow margin-top is creating a spacing between rows
+  margin-bottom: -9px;
```

Comment:
Hot-fix in the sense that this whole table will be deprecated soon, right?

Reply:
@josephkmh Yes, in the sense that fixing the root issue is not as simple as adjusting the margins in the StreamHeader and that this table is on its way out in the next few months.